boot/freeldr/freeldr/arch/uefi/ueficon.c

```diff
@@ -138,6 +138,5 @@ UefiConsGetCh(VOID)
     /* UEFI will stack input requests, we have to clear it */
     Key.UnicodeChar = 0;
     Key.ScanCode = 0;
-    GlobalSystemTable->ConIn->Reset(GlobalSystemTable->ConIn, FALSE);
```

Comment:
What does this removal fix?

Reply:
One some hardware I’ve found this isn’t fully supported as an operation so it’s solely just to lower minimal requirements,

also it’s a bit buggy in virtualbox (sometimes it makes long and press not register